sdk/compute/Azure.ResourceManager.Compute/tests/Mock/MockingTests.cs

```diff
@@ -95,7 +95,7 @@ public async Task Mocking_GetCollectionAndCreate()
             var setLro = await setCollection.CreateOrUpdateAsync(WaitUntil.Completed, setName, setData);
```

Comment:
we could add `default` here as well just in case the same thing happens to this method in the future.
```suggestion
            var setLro = await setCollection.CreateOrUpdateAsync(WaitUntil.Completed, setName, setData, default);
```